### PR TITLE
fixes #9033: errors encountered in new container wizard are being shown now

### PR DESCRIPTION
**Pull request by witlessbird**

### CODE REVIEW

**Comment by elyezer** [comment]
Looking forward it, thank you.

File: app/models/service/containers.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 module Service
   class Containers
-    def self.start_container!(wizard_state)
+    def errors
```
Comment (eLobato) [inline app/models/service/containers.rb]: I'm wondering if errors should be here or in the DockerContainerWizardState itself @witlessbird ?

Comment (witlessbird) [inline app/models/service/containers.rb]: I added two validations to configuration step; we should do the same to other steps when possible (fail fast and all that). Remaining errors are probably more related to the service side of things, methinks.

**Comment by eLobato** [comment]
Merged as 63ae5d99f7873fea396feee1ccbadc9c476805c9, thanks @witlessbird . I merged this as soon as I could because it'll make other PRs to rebase

**Pull request closed without merging**